src/bioregistry/data/curated_papers.tsv

```diff
@@ -1,4 +1,7 @@
 pubmed	relevant	orcid	date_curated	relevancy_type	pr_added	notes
+26473382	1	0009-0008-8406-631X	2025-2-1	new_publication	1394	Publication for mirgenedb
+31598695	1	0009-0008-8406-631X	2025-2-1	new_publication	1394	Publication for mirgenedb
+34850127	1	0009-0008-8406-631X	2025-2-1	new_publication	1394	Publication for mirgenedb
```

Comment:
The three old papers that didn't come out of the prediction pipeline can be removed from this file